src/main/kotlin/no/nav/amt/deltaker/bff/deltaker/DeltakerService.kt

```diff
@@ -36,34 +33,31 @@ class DeltakerService(
         personident: String,
         opprettetAv: String,
         opprettetAvEnhet: String?,
-    ): DeltakerResponse {
-        val deltakerliste = deltakerlisteRepository.get(deltakerlisteId)
-            ?: throw NoSuchElementException("Fant ikke deltakerliste med id $deltakerlisteId")
-        val eksisterendeDeltaker = deltakerRepository.get(personident, deltakerlisteId)
+    ): Deltaker {
+        val deltakerliste = deltakerlisteRepository.get(deltakerlisteId).getOrThrow()
```

Comment:
Hva er det som logges egentlig med getOrThrow? Det er veldig fint å kunne logge id på den deltakerlisten som ikke fantes, hvis ikke har vi ingenting å gå etter hvis denne feilen skjer i prod en dag :)

Reply:
Sånn koden var ble det logget av exception handleren lenger ut (selv om det beste er å logge direkte i koden der feilen skjer). Så da blir vel kanskje spørsmålet heller: Hva slags exception kastes her, og blir den håndtert riktig i exception handleren? :)

Reply:
Fant det lenger ned :) Da burde det oppføre seg likt som før ja. Synes koden blir litt vanskeligere å lese når man bruker Result, men det er sikkert en vanesak.

Reply:
Det er nok litt personlig preferanse ja, jeg har ikke så mye i mot det som det var men det blir altid så man må kaste nye exceptions hele tiden, isteden for å definere ett. Man kan fortsatt kaste unike exceptions hvis det gir mening å logge noe annet enn det som er der fra før.